popup.js

```diff
@@ -123,6 +123,24 @@ async function launchScript(shuffle = false) {
     const title = titleInput.value
     const channelName = channelNameInput.value
 
+    const css = `
+    @keyframes jump {
+        0% {
+            transform: scale(1);
+        }
+        100% {
+            transform: scale(1.02);
+        }
+    }
+    `;
```

Comment:
move this variable outside this function, it's useless and long here (or in function) ?